### PR TITLE
[WIP] Disabled shared compilation

### PR DESCRIPTION
**Pull request by distantcam**

### CODE REVIEW

File: src/ServiceInsight/ServiceInsight.csproj [hunk 1]
```diff
@@ -45,9 +46,6 @@
     <UseVSHostingProcess>false</UseVSHostingProcess>
     <CodeAnalysisRuleSet>..\CommonRuleSet.ruleset</CodeAnalysisRuleSet>
   </PropertyGroup>
-  <PropertyGroup>
-    <ApplicationIcon>ServiceInsight.ico</ApplicationIcon>
```
Comment (HEskandari) [inline src/ServiceInsight/ServiceInsight.csproj]: Was this intentionally removed?

File: src/ServiceInsight/ServiceInsight.csproj [hunk 2]
```diff
@@ -21,6 +21,7 @@
     <NuGetPackageImportStamp>
     </NuGetPackageImportStamp>
     <ExpressionBlendVersion>12.0.50429.0</ExpressionBlendVersion>
+    <UseSharedCompilation>false</UseSharedCompilation>
```
Comment (HEskandari) [inline src/ServiceInsight/ServiceInsight.csproj]: Is this disabled only for Debug config? How about Release?

File: src/ServiceInsight/ServiceInsight.csproj (hunk 1 above)
Comment (distantcam) [inline src/ServiceInsight/ServiceInsight.csproj]: It's already set in a previous property group. This is redundant.

File: src/ServiceInsight/ServiceInsight.csproj (hunk 2 above)
Comment (distantcam) [inline src/ServiceInsight/ServiceInsight.csproj]: It's disabled for both.

**Comment by distantcam** [comment]
@Particular/serviceinsight-maintainers This was to test if this would fix some errors occurring in the build. Instead the build was fixed with a parameter so this PR is no longer needed. Closing.

**Pull request closed without merging**